reconcile/gitlab_housekeeping.py

```diff
@@ -19,6 +19,26 @@
 
 QONTRACT_INTEGRATION = 'gitlab-housekeeping'
 
+def clean_pipelines(dry_run, gl, pipeline_timeout=60):
+    DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+    now = datetime.utcnow()
+
+    pipelines = gl.get_pipelines(status='pending')
+
+    for pipeline in pipelines:
```

Comment:
You'll see me complain about loop variables that differ from the iterated object by just one character. Try making sense of them on long loops after 6 hours of work and interruptions...
```suggestion
    for p in pipelines:
```

Reply:
In my understanding variables with one letter names are bad for readability, but if this is a project standard I can change that.

Reply:
Well, that depends on the scope. The adagio is "use short names for locals, long for globals". You don't want a single-letter class name or method name. But for a very short-lived loop variable, whose "main" object of interest is 3 characters away, the shorter the better. Also, if this loop ever grows, you'll *really* mistake `pipeline` and `pipeline` (and I've seen the one different character placed in the middle of the word, try to read that).

Longer discussion: https://gitlab.cee.redhat.com/service/dev-guidelines/-/merge_requests/63 .